### PR TITLE
Update postgres images, make sure all db services have versions in them

### PR DESCRIPTION
**Pull request by derschatta**
Addresses #159

This requires a few steps when you are using the affected services (mysql, mssql, pgsql, mariadb):

1. Stop the container, e.g. `tstop mysql`
2. Remove the old container, e.g. `docker container rm totara_mysql57`
3. Pull the changes
4. Then start the container with `tdocker up -d --remove-orphans mysql57`

If you follow these steps the data should still be there. Of course you need to update your config.php to connect to `mysql57` instead of `mysql`.

### CODE REVIEW

**Review by NathanielWalmsley — APPROVED** [review]
Tested with PGSQL 13 - all data was preserved, able to take down and re-up with no issues

**Comment by derschatta** [comment]
Thanks @NathanielWalmsley but the container we renamed is pgsql -> pgsql12. Probably best to try that one too.

**Comment by derschatta** [comment]
rebased path as well

File: php/php80/Dockerfile-alpine [hunk 1]
```diff
@@ -0,0 +1,130 @@
+FROM php:8.0-fpm-buster
```
Comment (NingZhou-NZ) [inline php/php80/Dockerfile-alpine:1]: `FROM php:8.0-fpm-alpine`

File: php/php80/Dockerfile-alpine [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        git \
+        libzip-dev \
+        libmemcached-dev \
+        zip \
+        netcat \
+        bc \
+        ghostscript \
+        graphviz \
+        aspell \
+        libldap2-dev \
```
Comment (NingZhou-NZ) [inline php/php80/Dockerfile-alpine:30]: I think this need to be `RUN apk add --no-cache`

File: php/php80/Dockerfile-buster [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+ENV LANGUAGE en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+
+# install mssql extension
+RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
+    && curl https://packages.microsoft.com/config/debian/10/prod.list > /etc/apt/sources.list.d/mssql-release.list \
+    && apt-get update && ACCEPT_EULA=Y apt-get install -y \
+    msodbcsql17 \
+    mssql-tools \
+    unixodbc-dev
```
Comment (NingZhou-NZ) [inline php/php80/Dockerfile-buster:84]: `RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
    && curl https://packages.microsoft.com/config/debian/11/prod.list > /etc/apt/sources.list.d/mssql-release.list \
    && apt-get update && ACCEPT_EULA=Y apt-get install -y \
    msodbcsql18 \
    mssql-tools18 \
    unixodbc-dev`

File: php/php80/Dockerfile-buster [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+# Workaround to get MSSQL connection working on Debian 10 (buster)
+# https://emacstragic.net/2017/11/06/mssql-odbc-client-on-debian-9-stretch/
+RUN wget "http://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.0.0_1.0.1t-1+deb8u12_amd64.deb" \
+    && DEBIAN_FRONTEND=noninteractive dpkg -i ./libssl1.0.0_1.0.1t-1+deb8u12_amd64.deb
```
Comment (NingZhou-NZ) [inline php/php80/Dockerfile-buster:89]: I think this can be removed

**Review by NingZhou-NZ — COMMENTED** [review]
All good. Just a few suggestions

File: php/php80/Dockerfile-alpine (hunk 1 above)
Comment (derschatta) [reply php/php80/Dockerfile-alpine:1]: I've already changed those to bullseye in #248 

Going to alpine requires a bit more work and testing and I think I will open a new PR for that.

File: php/php80/Dockerfile-buster (hunk 4 above)
Comment (derschatta) [reply php/php80/Dockerfile-buster:89]: yep that's already done in the `develop` branch. see #244

File: php/php80/Dockerfile-buster (hunk 3 above)
Comment (derschatta) [reply php/php80/Dockerfile-buster:84]: already covered in #244

File: php/php80/Dockerfile-alpine (hunk 2 above)
Comment (derschatta) [reply php/php80/Dockerfile-alpine:30]: related to alpine, that's going to be a separate piece of work for the php images

**Pull request merged**